docs/website/docs/plus/getting-started/transformations.md

```diff
@@ -0,0 +1,446 @@
+    employee_tables = dataset.employees
+    payroll_table = dataset.payroll
+    joined_table = employee_tables.join(payroll_table, employee_tables.id == payroll_table.employee_id)
+    # we select the relevant columns, making department and office id directly available
+    return joined_table.select(
+        payroll_table.id,
+        payroll_table.employee_id,
+        payroll_table.month,
+        payroll_table.amount,
+        employee_tables.department_id,
+        employee_tables.office_id
+    )
+
+@dlt.source()
+def star_source(dataset: SupportsReadableDataset):
+    return payroll_facts(dataset),employees(dataset), departments(dataset), offices(dataset)
+
+
+# create the star schema in the star warehouse
+info = star_pipeline.run(star_source(company_dataset))
+print(info)
+
+# now that we have a nice star schema, presumable in a column oriented warehouse, we can aggregate
+# payments per department with only a single join:
+star_dataset = star_pipeline.dataset()
+fact_table = star_dataset.payroll_facts
+office_table = star_dataset.offices
+
+joined_table = fact_table.join(office_table, fact_table.office_id == office_table.id)
+print(joined_table.group_by(office_table.city).aggregate(total_payments=joined_table.amount.sum()).arrow())
+
+
+```
+
+:::tip
+For simply copying table data, you can also use the sync source demonstrated below. Also note that in future versions of the dlt+ transformation framework, there will be a template generator to create a star schema from a given dataset to give you a starting point.
+:::
+
+
+## Lineage
+
+dlt+ transformations contain a lineage engine that can trace the origin of columns resulting from transformations. You may have noticed that we added a custom hint to the name column in the employees table at the beginning of the page. This hint is a custom hint that we decided to add to all columns containing very sensitive data. Ideally, we would like to know which columns in a result are derived from columns containing sensitive data. dlt+ lineage will do just that for you. Let's run an aggregated join query into our warehouse again, but this time we will not drop the name column:
+
+```python
+
+@dlt_plus.transform(transformation_type="python")
+def payment_per_employee(dataset: SupportsReadableDataset):
+    employee_tables = dataset.employees
+    payroll_table = dataset.payroll
+    joined_table = employee_tables.join(payroll_table, employee_tables.id == payroll_table.employee_id)
+    return joined_table.group_by(joined_table.employee_id, joined_table.name).aggregate(total_payments=joined_table.amount.sum())
+
+# run the transformation
+info = warehouse_pipeline.run(payment_per_employee(company_dataset))
+print(info)
+
+```
+
+You can now inspect the schema of the warehouse and see that the name column of the aggregated table is also marked with our hint: `uv run dlt pipeline warehouse_pipeline schema`. dlt+ uses the same mechanism to derive precision and scale information for columns.
```

Comment:
> . dlt+ uses the same mechanism to derive precision and scale information for columns.

its not clear to me what that means? Are numbers (e.g. in financial applications) affected by precision changes? what does scale mean here?